### PR TITLE
Add pages

### PR DESCRIPTION
**Pull request by Istanful**

### CODE REVIEW

File: app/models/social_media.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class SocialMedia
+  attr_reader :title
+  attr_reader :fa_i
```
Comment (alex-ross) [inline app/models/social_media.rb:3]: Why does `fa_i` stands for?

File: app/views/pages/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <article>
+    <h2>Idéen av appen</h2>
+    <p>
+      Med hjälp av Sorteramera kan du hitta din närmsta återvinningsstation eller återvinningscentral
+      och även få vägbeskrivning dit. För varje återvinningsstation kan du får reda på information
+      om vilka material som går att återvinna, vilken adress eller ort den finns på och även vilka öppettider stationen eller centralen har.
+    </p>
+
+    <img class="screenshot">
+    </img>
```
Comment (alex-ross) [inline app/views/pages/index.html.erb]: You don't need a close tag for `img`-tags in HTML5.

File: app/views/pages/api.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <h2>Nu även ett API!</h2>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt
+      ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+      quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+      Besök <a href="#">dokumentationen</a> för att veta mer!
+    </p>
+    <img class="screenshot">
+    </img>
```
Comment (alex-ross) [inline app/views/pages/api.html.erb]: You don't need a close tag for `img`-tags in HTML5.

File: app/views/pages/api.html.erb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+<main>
+  <article>
+    <h2>Nu även ett API!</h2>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt
+      ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+      quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+      Besök <a href="#">dokumentationen</a> för att veta mer!
```
Comment (alex-ross) [inline app/views/pages/api.html.erb]: I like to have `FIXME` or `TODO` comments when something is not done. In this case we may miss to add the documentation href later. But if you add a issue in the task manager for the project that will do as well.

**Review by alex-ross — COMMENTED** [review]
Well done. Is the image `pexels-photo-90807.jpeg` used anywhere?

File: app/models/social_media.rb (hunk 1 above)
Comment (Istanful) [reply app/models/social_media.rb:3]: Font awesome icon tag.

File: app/views/pages/index.html.erb (hunk 2 above)
Comment (Istanful) [reply app/views/pages/index.html.erb]: Whoops. Fixing!

File: app/views/pages/api.html.erb [hunk 5]
```diff
@@ -11,9 +11,8 @@
       ut labore et dolore magna aliqua. Ut enim ad minim veniam,
       quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
 
-      Besök <a href="#">dokumentationen</a> för att veta mer!
+      Besök <a href="#">dokumentationen</a> för att veta mer!                                               <!-- TODO -->
```
Comment (alex-ross) [inline app/views/pages/api.html.erb]: Good, but not what i expected. A todo should always describe what you expect the next person, or yourself in the future to do.

Then we always know later what is expected and if no-one removes the TODO after fixing the issue we also know when it's outdated.

Example.

``` html
<!-- TODO: Fill in the `href` with the correct URL when that page exists. -->
Besök <a href="#" ...
```

**Review by alex-ross — CHANGES_REQUESTED** [review]

File: app/models/social_media.rb (hunk 1 above)
Comment (alex-ross) [reply app/models/social_media.rb:3]: Ok, great :) Then I know :)

**Pull request merged**